corehq/apps/hqwebapp/management/commands/build_bootstrap5_diffs.py

```diff
@@ -147,7 +146,6 @@ def handle(self, *args, **options):
             self.stdout.write(self.style.ERROR(
                 "You have un-committed changes. Please commit these changes before proceeding...\n"
```

Comment:
Is this message still relevant/correct?

Reply:
Yes. It's still true, and it's still not a bad idea to commit unstaged changes, we're just no longer forcing the user to do so.